src/components/MyKiva/ImpactInsight/MyKivaImpactInsightScreen1.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+	<section class="tw-w-full tw-px-2	 md:!tw-py-10 md:!tw-px-10">
+		<div class="tw-mx-auto tw-max-w-6xl">
+			<!-- Title -->
+			<h2 class="">
+				A closer look at <u>{{ borrowerName }} world</u>
+			</h2>
+			<div
+				class="tw-mt-10 tw-flex tw-flex-col tw-items-center tw-gap-10
+						md:tw-mt-14 md:tw-flex-row md:tw-items-center md:!tw-justify-center"
+			>
+				<div
+					class="tw-flex tw-items-center tw-justify-center tw-rounded-full tw-border-4 tw-border-brand-500
+							tw-bg-eco-green-1 tw-text-center min-circle-size"
+				>
+					<div class="tw-items-center tw-gap-3">
+						<div
+							class="tw-h-6.5 tw-w-6.5 tw-overflow-hidden tw-rounded-full
+									tw-border-4 tw-border-white tw-my-0 tw-mx-auto"
+						>
+							<KvBorrowerImage
+								class="tw-w-full tw-h-full tw-object-cover"
+								:alt="name"
+								:hash="hash"
+								:aspect-ratio="2 / 2"
+								:default-image="{ width: 60 }"
+								:images="[{ width: 60 }]"
+								:photo-path="$appConfig.photoPath"
+							/>
+						</div>
+
+						<div class="tw-text-lg tw-font-semibold tw-text-slate-900">
+							<strong>{{ name }}</strong>
+						</div>
+					</div>
+				</div>
+				<div
+					class="tw-w-full tw-max-w-xl tw-px-4 md:!tw-px-0"
+				>
+					<p
+						class="tw-rounded-2xl tw-bg-slate-100 tw-py-2 tw-px-2 md:!tw-px-3
+							tw-text-base tw-leading-relaxed tw-bg-gray-100 tw-rounded-md md:tw-text-lg"
+					>
+						Your $25 loan helps {{ name }} build stability and success in
+						<strong class="tw-text-brand">{{ countryName }}</strong>,
+						where the average
+						<strong class="tw-text-brand">annual income is $2,700 USD</strong>.
```

Comment:
This annual income needs new monolith BE work, if the data isn't already being returned. Do we have a ticket under the same epic for this part of the work? I forget if we created one.